src/components/Table/TableFormatters/DetailFormatter.vue

```diff
@@ -114,16 +114,16 @@ export default {
       return resource.replace(' details', '').replace('详情', '')
     },
     getDrawerTitle() {
+      let title = this.cellValue || this.row.name
+
       if (this.formatterArgs?.getDrawerTitle && typeof this.formatterArgs.getDrawerTitle === 'function') {
-        this.formatterArgs.getDrawerTitle({
+        title = this.formatterArgs.getDrawerTitle({
           col: this.col,
           row: this.row,
           cellValue: this.cellValue
         })
       }
 
-      let title = this.cellValue || this.row.name
-
       const resource = this.getResource()
 
       if (resource) {
```

Comment:
There doesn't seem to be any known issues with this code snippet, it seems very basic and straightforward. In terms of optimization, you could consider making the `title` variable local instead of using `col`, `row` or `cellValue` inside the formatter argument method (`this.formatterArgs.getDrawerTitle({})`). The current use of "col" is not necessary because it's already used in your resource path.

Regarding differences between previous versions (October 2021), there aren't clear changes indicated beyond these few lines that don't significantly impact readability/efficiency but which can potentially save on resources by improving performance.

If anything more complex needs checking please ask, I'm here for any clarification!